uni/lib/view/Widgets/course_info_card.dart

```diff
@@ -5,11 +5,9 @@ import 'package:uni/view/Widgets/generic_card.dart';
 /// Manages the courses info (course name, atual year, state and year of
 /// first enrolment) on the user personal page.
 class CourseInfoCard extends GenericCard {
```

Comment:
If you decide to implement multiples courses, clicking in a course card here could go to the UCS's page

Reply:
I would prefer to leave that to another issue, as that requires relating the course unit to the course on the model level